src/VirtoCommerce.Platform.Core/ChangeLog/ChangeLogSearchCriteria.cs

```diff
@@ -5,6 +5,7 @@ namespace VirtoCommerce.Platform.Core.ChangeLog
 {
     public class ChangeLogSearchCriteria : SearchCriteriaBase
     {
+        public EntryState? OperationType { get; set; } 
```

Comment:
I think it is better to add EntryState[] property and add conditions only if it is not empty.